pkg/virt-handler/vm.go

```diff
@@ -58,6 +58,8 @@ import (
 	"kubevirt.io/kubevirt/pkg/watchdog"
 )
 
+var MigrationPortsRange = []int{49152, 49153}
```

Comment:
I think you can get it from `migration-proxy` package.

Reply:
yeah, this one doesn't appear to be used anywhere right now.

Reply:
yeah, thanks, forgot about it.